app/src/main/java/com/yml/womensafety/viewModel/SafetyTipsViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.yml.womensafety.viewModel
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.yml.womensafety.navigationdrawer.SafetyTips
+import com.yml.womensafety.repository.SafetyTipsRepository
+
+class SafetyTipsViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
Why are we passing the `application` here?

Reply:
I used it to access string in strings.xml

Reply:
Removed it.